.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,51 +1,94 @@
 name: 🐞 Bug Report
-description: File a new bug report
-title: 'bug: <title>'
+description: Create a new bug report.
+title: '🐞 bug: <title>'
 labels: [Bug, Needs Triage]
 body:
+
   - type: markdown
     attributes:
-      value: ':stop_sign: _For support questions, please visit [http://discuss.appium.io](http://discuss.appium.io) instead._'
+      value: ':stop_sign: _Make sure you are reporting an issue. For general questions, please use [http://discuss.appium.io](http://discuss.appium.io)._'
+
   - type: checkboxes
     attributes:
-      label: 'Do I have the most recent component updates?'
-      description: 'Appium 1-related issues are going to be ignored as this version has reached its end of life.'
+      label: Is there an existing issue for this?
+      description: 'Please [search :mag: the issues](https://github.com/appium/appium/issues) to check if this bug has already been reported.'
       options:
-      - label: 'I use the most recent available driver/plugin and server versions'
+      - label: I have searched the existing issues
         required: true
-  - type: checkboxes
+  
+  - type: input
     attributes:
-      label: 'Is the component officially supported by the Appium team?'
-      description: 'Make sure you report an issue for a component which is officially supported/maintained by the Appium team.'
-      options:
-      - label: 'I have verified the component repository is present under the Appium organization in GitHub'
-        required: true
-  - type: checkboxes
+      label: Appium Version
```

Comment:
Perhaps it is a good time to simplify that and ask people to just provide the output of `appium server --show-debug-info` (with sensitive data sanitized)